tardis/apps/publication_workflow/components/PublicationsHome.jsx

```diff
@@ -0,0 +1,103 @@
+import React, {
+  Fragment, useEffect, useState,
+} from 'react';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
} from "react";
```